tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -437,12 +405,7 @@ public Dictionary<string, object> GetBasicRequestArgsForWaf()
             }
         }
 
-        var dict = new Dictionary<string, object>(capacity: 7)
-        {
-            { AddressesConstants.RequestMethod, request.HttpMethod },
-            { AddressesConstants.ResponseStatus, request.RequestContext.HttpContext.Response.StatusCode.ToString() },
-            { AddressesConstants.RequestClientIp, _localRootSpan.GetTag(Tags.HttpClientIp) }
-        };
+        var dict = new Dictionary<string, object>(capacity: 7) { { AddressesConstants.RequestMethod, request.HttpMethod }, { AddressesConstants.ResponseStatus, request.RequestContext.HttpContext.Response.StatusCode.ToString() }, { AddressesConstants.RequestClientIp, _localRootSpan.GetTag(Tags.HttpClientIp) } };
```

Comment:
I suppose it's auto-formating that changed this line and no you directly, but I preferred the previous version.

Reply:
I agree, and I'm not sure why it shows like that, cause it's still wrapped in the editor 🤔 
![image](https://github.com/user-attachments/assets/3f3794ad-fe9c-4257-9b25-fc6cc2bc8541)